src/Testcontainers.Papercut/PapercutBuilder.cs

```diff
@@ -0,0 +1,71 @@
+using System.Net;
+
+namespace Testcontainers.Papercut;
+
+/// <inheritdoc cref="ContainerBuilder{TBuilderEntity, TContainerEntity, TConfigurationEntity}" />
+[PublicAPI]
+public sealed class PapercutBuilder : ContainerBuilder<PapercutBuilder, PapercutContainer, PapercutConfiguration>
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="Testcontainers.PapercutBuilder" /> class.
+    /// </summary>
+    public PapercutBuilder()
+        : this(new PapercutConfiguration())
+    {
+        DockerResourceConfiguration = Init().DockerResourceConfiguration;
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="Testcontainers.PapercutBuilder" /> class.
+    /// </summary>
+    /// <param name="resourceConfiguration">The Docker resource configuration.</param>
+    private PapercutBuilder(PapercutConfiguration resourceConfiguration)
+        : base(resourceConfiguration)
+    {
+        DockerResourceConfiguration = resourceConfiguration;
+    }
+
+    /// <inheritdoc />
+    protected override PapercutConfiguration DockerResourceConfiguration { get; }
+
+    /// <inheritdoc />
+    public override PapercutContainer Build()
+    {
+        Validate();
+        return new PapercutContainer(DockerResourceConfiguration, TestcontainersSettings.Logger);
+    }
+
+    /// <inheritdoc />
+    protected override PapercutBuilder Init()
+    {
+        return base.Init()
+            .WithImage("jijiechen/papercut:latest")
+            .WithPortBinding(25,true)
+            .WithPortBinding(37408,true)
```

Comment:
Please use a const and a meaningful name that describes the port.

Reply:
Done